lib/matplotlib/tests/test_colorbar.py

```diff
@@ -913,3 +914,19 @@ def test_negative_boundarynorm():
     cb = fig.colorbar(cm.ScalarMappable(cmap=cmap, norm=norm), cax=ax)
     np.testing.assert_allclose(cb.ax.get_ylim(), [clevs[0], clevs[-1]])
     np.testing.assert_allclose(cb.ax.get_yticks(), clevs)
+
+
+@image_comparison(['nonorm_colorbars.svg'], remove_text=False,
+                  style='mpl20')
+def test_nonorm():
+    data = [1, 2, 3, 4, 5]
+
+    fig, ax = plt.subplots(figsize=(6, 1))
+    fig.subplots_adjust(bottom=0.5)
+
+    norm = NoNorm(vmin=min(data), vmax=max(data))
+    cmap = cm.get_cmap("viridis", len(data))
+    mappable = cm.ScalarMappable(norm=norm, cmap=cmap)
+    cbar = fig.colorbar(mappable, cax=ax, orientation="horizontal")
+    fig.savefig('/tmp/wat.svg')
+    fig.savefig('/tmp/wat.png')
```

Comment:
```suggestion
```

Reply:
🤦🏻 At least some of my even more frustrated debugging statements did not make it in!